src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskLoaderImpl.scala

```diff
@@ -0,0 +1,32 @@
+package mesosphere.marathon.core.task.tracker.impl
+
+import mesosphere.marathon.state.TaskRepository
+import mesosphere.marathon.tasks.TaskIdUtil
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.Future
+
+/**
+  * Loads all task data into an [[AppDataMap]] from a [[TaskRepository]].
+  */
+private[tracker] class TaskLoaderImpl(repo: TaskRepository) extends TaskLoader {
+  import scala.concurrent.ExecutionContext.Implicits.global
+
+  private[this] val log = LoggerFactory.getLogger(getClass.getName)
+
+  override def loadTasks(): Future[AppDataMap] = {
+    for {
+      names <- repo.allIds()
+      _ = log.info(s"About to load ${names.size} tasks")
```

Comment:
It is displayed once on startup. I would keep it at info. You do see if Marathon was stuck loading the tasks that way.